cfgov/unprocessed/js/modules/util/dom-events.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Shortcut for binding event listeners to elements
+ * @param  {HTMLNode} elem   The element to attach the event listener to
+ * @param  {Object}   events The list of events to attach to the element
+ */
+function bindEvent( elem, events ) {
```

Comment:
I think it would be useful to provide a parameter that will allow us bind do a certain context other than the element. We often will do `element.addEventListener( 'click', _function.bind( this ) );`